### PR TITLE
[SLWP] Kands further text improvements

### PR DESCRIPTION
**Pull request by MorayMySoc**
Changes from https://3.basecamp.com/4020879/buckets/33314444/todos/6641263357 and (https://3.basecamp.com/4020879/buckets/33314444/todos/6618688024 & https://3.basecamp.com/4020879/buckets/33314444/todos/6618280655)

[skip changelog]

### CODE REVIEW

File: perllib/FixMyStreet/App/Form/Page/Waste.pm [hunk 1]
```diff
@@ -6,11 +6,21 @@ extends 'FixMyStreet::App::Form::Page::Simple';
 has title => ( is => 'ro', 'isa' => 'Str', lazy => 1, builder => '_build_title' );
 # So we can insert cobrand specific service name
 has title_ggw => ( is => 'ro', isa => 'Str' );
+has bulky_general_title => ( is => 'ro', 'isa' => 'Str' );
```
Comment (dracos) [inline perllib/FixMyStreet/App/Form/Page/Waste.pm:9]: bulky_general_title isn't actually used, it looks like?

File: perllib/FixMyStreet/App/Form/Page/Waste.pm [hunk 2]
```diff
@@ -6,11 +6,21 @@ extends 'FixMyStreet::App::Form::Page::Simple';
...
 
 sub _build_title {
     my $self = shift;
-    my $name = $self->form->{c}->cobrand->garden_service_name;
-    return sprintf($self->title_ggw, $name);
+    if ($self->form =~ /Bulky/) {
```
Comment (dracos) [inline perllib/FixMyStreet/App/Form/Page/Waste.pm:13]: Perhaps could subclass Form/Page/Waste.pm and override _build_title there? `default_page_type` on Form/Waste/Bulky.pm selects which "Page" class to use.

File: templates/web/base/waste/bulky/confirmation.html [hunk 3]
```diff
@@ -15,7 +15,13 @@ <h1 class="govuk-panel__title">
   </h1>
   <div class="govuk-panel__body">
       <p>Our contractor will collect the items you have requested on [% cobrand.bulky_nice_collection_date(report) %].</p>
-      <p>Item collection starts from [% cobrand.bulky_nice_collection_time %]. Please have your items ready and dismantled if required.</p>
+      <p>Item collection starts from [% cobrand.bulky_nice_collection_time %].&nbsp;
```
Comment (dracos) [inline templates/web/base/waste/bulky/confirmation.html:18]: No need to change, I just wondered why it was switched to a nbsp?

File: perllib/FixMyStreet/App/Form/Page/Waste.pm (hunk 1 above)
Comment (MorayMySoc) [reply perllib/FixMyStreet/App/Form/Page/Waste.pm:9]: It's used in perllib/FixMyStreet/App/Form/Waste/Bulky.pm which is how I thought it took effect though I was kind of muddling through

File: templates/web/base/waste/bulky/confirmation.html (hunk 3 above)
Comment (MorayMySoc) [reply templates/web/base/waste/bulky/confirmation.html:18]: I thought the removal of spaces with ```[%~``` would mean there was no space after the full stop

File: perllib/FixMyStreet/App/Form/Page/Waste.pm (hunk 1 above)
Comment (dracos) [reply perllib/FixMyStreet/App/Form/Page/Waste.pm:9]: Where is it used? I see you've added some blank entries for it, but they're not actually then used anywhere?

File: templates/web/base/waste/bulky/confirmation.html (hunk 3 above)
Comment (dracos) [reply templates/web/base/waste/bulky/confirmation.html:18]: Yeah, but it's HTML, you didn't have to use `[%~` :)

File: perllib/FixMyStreet/App/Form/Page/Waste.pm (hunk 1 above)
Comment (MorayMySoc) [reply perllib/FixMyStreet/App/Form/Page/Waste.pm]: Yeah, was blundering about in the dark a bit there!

**Review by dracos — APPROVED** [review]
Thanks, nice and neat! Fix the indenting in the new Page file before you merge :)

**Pull request merged**